prc/bls/circuit_selectors.tex

```diff
@@ -93,20 +93,21 @@
         we impose
         \[
             \left\{ \begin{array}{lcrcl}
-                \csPointEvaluation _{i} & = & \wellformedDataNontrivial _{i} & \cdot & \locIsPointEvaluation                    _{i} \\
-                \csGOneAdd         _{i} & = & \wellformedDataNontrivial _{i} & \cdot & \locIsBlsGOneAdd                         _{i} \\
-                \csGOneMsm         _{i} & = & \wellformedDataNontrivial _{i} & \cdot & \locIsBlsGOneMsm                         _{i} \\
-                \csGTwoAdd         _{i} & = & \wellformedDataNontrivial _{i} & \cdot & \locIsBlsGTwoAdd                         _{i} \\
-                \csGTwoMsm         _{i} & = & \wellformedDataNontrivial _{i} & \cdot & \locIsBlsGTwoMsm                         _{i} \\
-                \csPairing         _{i} & = & \wellformedDataNontrivial _{i} & \cdot & \locIsBlsIsNontrivialPairingDataOrResult _{i} \\
-                \csMapFpToGOne     _{i} & = & \wellformedDataNontrivial _{i} & \cdot & \locIsBlsMapFpToGOne                     _{i} \\
-                \csMapFpTwoToGTwo  _{i} & = & \wellformedDataNontrivial _{i} & \cdot & \locIsBlsMapFpTwoToGTwo                  _{i} \\
+                \csPointEvaluation _{i}        & = & \wellformedDataNontrivial _{i} & \cdot & \locIsPointEvaluation                    _{i} \\
+                \csPointEvaluationFailure _{i} & = & \malformedDataExternalTot _{i} & \cdot & \locIsPointEvaluation                    _{i} \\
```

Comment:
@ivokub How much detail do you want wrt to `POINTEVALUATION` failure ? The current approach of @lorenzogentile404 is to have two cases:
- (predicted to) succeed
- (predicted to) fail for an unspecified "external" reason

Reply:
It is sufficient to have a single failure reason - we will hint the actual exact failure reason(s) within a gnark circuit. I think we have more insight into the actual reason than the tracer would have.